readme.md

```diff
@@ -190,6 +190,56 @@ if (additionalPosts.length) {
 }
 ```
 
+
+#### 📍 no-implicit-coercion
+Encourages stopping mixing different types of variables for the sake of cleaner and more readable code.
+
+##### ❌ Example of incorrect code for this rule:
+
+```
+// Boolean
+const b = !!foo;
+const b = ~foo.indexOf(".");
+
+// Number
+const n = +foo;
+const n = 1 * foo;
+
+// Strings
+const s = "" + foo;
+const s = `` + foo;
+foo += "";
+foo += ``;
+
+```
+
+##### ✅ Example of correct code for this rule:
+
+```
+// Boolean
+const b = Boolean(foo);
+const b = foo.indexOf(".") !== -1;
```

Comment:
Would probably change to `.includes('.');` Also make sure we're using single quotes as opposed to double quotes